Example/package.json

```diff
@@ -95,7 +95,7 @@
         "react-native-simple-popover": "git+https://github.com/tonlabs/react-native-simple-popover.git",
         "react-native-status-bar-height": "1.0.1",
         "react-native-svg": "^12.1.0",
-        "react-native-video": "^5.2.0",
+        "react-native-video": "git+https://github.com/r0b0t3d/react-native-video.git#feature/android-fullscreen",
```

Comment:
Why so? This is an old repo which hasn't been updated for years.... :(

Reply:
It's all for fullscreen mode on android. This is a critical feature

Reply:
I changed it to `tonlabs/react-native-video`